designcompose/src/main/java/com/android/designcompose/DesignText.kt

```diff
@@ -357,9 +363,33 @@ internal fun DesignText(
                     }
                 )
             } else {
+                var textLayoutResult: TextLayoutResult? = null
                 BasicText(
                     annotatedText,
-                    modifier = textModifier,
+                    onTextLayout = { result ->
+                        textLayoutResult = result
```

Comment:
Oh, I didn't know we could get text layouts like this -- maybe this would also let us support text shadows better, too!

Reply:
So you have to wait for the text layout result here, and then use that in the modifier below? That's confusing to me.

Reply:
I think text layout happens before the draw modifier is executed, so there should always be a valid layout. It might be worth logging if there isn't (although perhaps empty strings don't generate a layout? Then the log could be annoying).

Reply:
In my understanding, this is how it works: 

When the text view is drawn on the canvas, it will measure -> layout -> draw.
- Then when it layouts, it will call the `onTextLayout` callback and we capture the `textLayoutResult`.
- Then when it draws, it calls the `Modifier.drawWithContent` and it can use the  `textLayoutResult` to draw the strokes.

I tried to set drawStyle to the TextStyle but it can only take one brush so it will not be able to draw the fill and stroke at the same time.
Other options is put two BasicText one on top of another. This solution is the best way I've gotten so far.